experiments/scripts/run_n_shot_benchmarks_eval.py

```diff
@@ -10,11 +10,19 @@
 CHEMNLP_FOLDER = "beth"
 DEFAULT_EVAL_CONFIG = (
     f"/fsx/proj-chemnlp/{CHEMNLP_FOLDER}"
-    "/chemnlp/experiments/configs/hugging-face/eval_configs/eval_pipeline_config.yaml"
+    "/fsx/proj-chemnlp/beth/chemnlp/experiments/configs/eval_configs/default_eval_config.yaml"
```

Comment:
We could use `CHEMNLP_FOLDER` inside of the `DEFAULT_EVAL_CONFIG` variable so there isn't the name beth in two places?

Reply:
Thanks for spotting that, the path was also wrong but it should be fixed now